app/views/product_notifier_mailer/new_product_available.text.erb

```diff
@@ -0,0 +1,3 @@
+New product available
+
+<%= @greeting %>, find me in app/views/product_notifier_mailer/new_product_available.text.erb
```

Comment:
What is this `find me in app/views/product_notifier_mailer/new_product_available.text.erb`?

Reply:
I removed it, it was a line I didn't need.